libs/block-content/website/src/lib/teaser/teaser.tsx

```diff
@@ -185,7 +217,68 @@ export const TeaserAuthors = styled('div')`
 
 export const TeaserContent = styled(TeaserInnerWrapper)``
 
+export const ImagePlaceholder = styled('div')`
+  width: 100%;
+  object-fit: cover;
+  grid-column: 1/13;
+  aspect-ratio: 1/1;
+`
+
+export const TeaserContentStyled = styled(TeaserContent)`
+  grid-column: 1/13;
+`
+
+export const TeaserPreTitleNoContent = styled(TeaserPreTitle)<{isHovered: boolean}>`
+  transition: background-color 0.3s ease-in-out;
+  background-color: ${({theme, isHovered}) =>
```

Comment:
dont think hover color style is part of the design

Reply:
hover stuff is taken from the note written on the designs, sent you screenshot on slack (uploading on gh doesn't work)